docs/layers/security-and-compliance/setup.mdx

```diff
@@ -1,284 +1,493 @@
 ---
 import Intro from '@site/src/components/Intro';
-import KeyPoints from '@site/src/components/KeyPoints'
-import Steps from '@site/src/components/Steps'
-import Step from '@site/src/components/Step'
-import StepNumber from '@site/src/components/StepNumber'
-import Admonition from '@theme/Admonition'
-import Note from '@site/src/components/Note'
+import KeyPoints from '@site/src/components/KeyPoints';
+import Steps from '@site/src/components/Steps';
+import Step from '@site/src/components/Step';
+import StepNumber from '@site/src/components/StepNumber';
+import Admonition from '@theme/Admonition';
+import Note from '@site/src/components/Note';
 import AtmosWorkflow from '@site/src/components/AtmosWorkflow';
 
 <Intro>
-These are the setup instructions for Security and Compliance in your AWS Organization.
+This guide walks you through deploying AWS security and compliance services across your AWS Organization.
+Follow the steps in order to ensure proper configuration of all services.
 </Intro>
 
-## Organization-Level Configuration
+## Prerequisites
 
-The following steps are required to set up Security and Compliance in your AWS Organization. They only be completed once for the entire organization.
+Before deploying security and compliance components, ensure:
+
+1. **AWS Organizations** is configured with your account structure
+2. **Account baseline** has been deployed (see [Deploy Accounts](/layers/accounts/deploy-accounts/))
+3. **SuperAdmin user** is available for root account deployments (see [How to Create SuperAdmin](/layers/accounts/tutorials/how-to-create-superadmin-user/))
+
+## Deployment Order
+
+The security components have dependencies and must be deployed in a specific order:
+
+```mermaid
+flowchart LR
+    subgraph phase1["Phase 1: Org-Level Config"]
+        p1["Service Principals<br/>Config Bucket<br/>CloudTrail Bucket"]
+    end
+
+    subgraph phase2["Phase 2: Foundational Services"]
+        p2["CloudTrail<br/>Config"]
+    end
+
+    subgraph phase3["Phase 3: Threat Detection"]
+        p3["GuardDuty<br/>Inspector<br/>Macie<br/>Access Analyzer"]
+    end
+
+    subgraph phase4["Phase 4: Aggregation & Protection"]
+        p4["Security Hub<br/>Shield<br/>Audit Manager"]
+    end
+
+    phase1 --> phase2 --> phase3 --> phase4
+```
+
+:::tip Workflow Recovery
+
+If a step in any workflow fails, you can restart from that failed step:
+
+```bash
+atmos workflow deploy/aws-config/global-collector -f compliance --from-step step4
+```
+
+:::
+
+---
+
+## Phase 1: Organization-Level Configuration
+
+These steps are required once for the entire organization.
 
 <Steps>
   <Step>
     ### <StepNumber/> Vendor Components
 
-    Vendor all data components with the following workflow:
+    Vendor all security and compliance components:
 
-    <AtmosWorkflow workflow="vendor" fileName="quickstart/monitor/security-and-compliance" />
+    <AtmosWorkflow workflow="vendor" fileName="compliance" />
```

Comment:
<img width="1680" height="956" alt="CleanShot 2025-12-26 at 13 11 41@2x" src="https://github.com/user-attachments/assets/87ee50fb-e8b1-439b-a3a2-e6518e8576f0" />

please double check the workflow file and command
https://github.com/cloudposse/docs/blob/master/examples/legacy/snippets/stacks/workflows/compliance.yaml